cyclonedx/schema/_res/bom-1.6.SNAPSHOT.xsd

```diff
@@ -7429,7 +7429,7 @@ limitations under the License.
                                 </xs:sequence>
                             </xs:complexType>
                         </xs:element>
-                        <xs:element name="cryptoRef" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="cryptoRefArray" type="bom:refType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
is this an intentional change?

you can not simply modify the schema here, it is downstream from 
 https://github.com/CycloneDX/specification/blob/master/schema/bom-1.6.xsd#L7432

Reply:
yeah i did that intentionally as i saw it was cryptoRefArray in json so thought it might also have to be same here